### ISSUE TITLE
Modify the contributing.md doc

### ISSUE THREAD

**Issue by michelleN** [kubernetes/helm#315]
We'll want to modify the contributing doc to include how community members can contribute to this project.
Include:
- process for contributing
- commit style guidelines
- testing guidelines 
- any other recommendations for successful contributions

etc.

**Comment by technosophos** [comment]
Can we just modify the existing one?
https://github.com/kubernetes/deployment-manager/blob/master/CONTRIBUTING.md

**Comment by michelleN** [comment]
yep. I'll change the title.

**Comment by jackgr** [comment]
@michelleN we followed the Kubernetes model, where CONTRIBUTING.md deals only with the CLAs, and another file, in this case the [README.md](https://github.com/kubernetes/deployment-manager#contributing), describes the process.

I'd prefer to stick with that model, if possible, since the CLAs are the single biggest issue we have faced historically, and keeping them in their own file calls attention to them. Fine to break out the process and other information from README.md, as long as we remove/relocate the existing content from README.md.
